lib/rspecq/reporter.rb

```diff
@@ -116,16 +116,18 @@ def humanize_duration(seconds)
     def flaky_jobs_to_sentry(jobs, build_duration)
       return if jobs.empty?
 
-      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
-        build: @build_id,
-        build_timeout: @timeout,
-        queue: @queue.inspect,
-        object: self.inspect,
-        pid: Process.pid,
-        flaky_jobs: jobs,
-        flaky_jobs_count: jobs.count,
-        build_duration: build_duration
-      })
+      jobs.each do |job|
+        Raven.capture_message("[Flaky] #{job}", level: "warning", extra: {
+          build: @build_id,
+          build_timeout: @timeout,
+          queue: @queue.inspect,
+          object: self.inspect,
+          pid: Process.pid,
+          path: job,
+          build_duration: build_duration,
+          flaky: true,
```

Comment:
I believe this is redundant, since we already have the respective tag. Is there any reason to have it here?